framework/src/fwk_status.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Arm SCP/MCP Software
+ * Copyright (c) 2019, Arm Limited and Contributors. All rights reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Description:
+ *     Framework status code helper.
+ */
+
+#include <fwk_status.h>
+#include <fwk_macros.h>
+
+const char *fwk_status_str(int status)
+{
+    switch (status) {
```

Comment:
What about to have two tables, one for the positive values and one for the negative/error values instead of this long switch?

Reply:
Ok. True that the array way looks better.